model_templates/training/python3_sklearn/push.yaml

```diff
@@ -0,0 +1,8 @@
+model:
+    name: drumpush-regression
+    type: training
+    targetType: regression
+    environmentID: 5e8c889607389fe0f466c72d
+client:
```

Comment:
Client part should not be part of the metadata since this is not related to the model  itself.
Probably the endpoint and  token should be  provided via separate arguments to the drum or env

drum push -d /opt/models/my_model_1 --endpoint XXXX --token YYYY

Reply:
and env ID also should go just to the CLI param

Reply:
drum push will create a model?
What will happen if I call `drum push` again? model will be existing model will be updated or another model will be  created?
Maybe we need to add model Id? And probably it should be just a string?

Reply:
> > why yaml? I think JSON might be more universally understood among non-engineers
> 
> Yaml is less ugly.

agree that yaml is more user friendly and readable